### PR TITLE
Added an initial PR template.

### PR DESCRIPTION
**Pull request by kshakir**

### CODE REVIEW

**Comment by ktibbett** [comment]
:+1:.  I've also asked @kbergin to take a look....

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  * Team may comment on PR at will
+  * **Reviewer assigns to submitter** for feedback fixes
+  * Submitter rebases to develop again if necessary
+  * Submitter makes further commits. DO NOT SQUASH
+  * Submitter updates documentation as needed
+  * Submitter **reassigns to reviewer** for further feedback
+- [ ] **Submitter**: Verify all tests go green, including CI tests
+- [ ] **Submitter**: Squash commits and merge to develop
+- [ ] **Submitter**: Delete branch after merge
+- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
```
Comment (kbergin) [inline .github/PULL_REQUEST_TEMPLATE.md:31]: Would there be another step here where after you test it on Dev you move it to staging and move ticket to QA?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+- [ ] **Submitter**: Include the JIRA issue number in the PR description
+- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
+- [ ] **Submitter**: If you're adding/switching libraries or otherwise changing the design, update the [design documentation](https://broadinstitute.atlassian.net/wiki/pages/viewpage.action?pageId=114531509).
```
Comment (kbergin) [inline .github/PULL_REQUEST_TEMPLATE.md]: Should there be a checkbox to add/include tests?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+- [ ] **Submitter**: Include the JIRA issue number in the PR description
```
Comment (kbergin) [inline .github/PULL_REQUEST_TEMPLATE.md:13]: I like how in GSA-unstable they first have a section for a Description (header) and then a section for a Checklist where this part would all go. Example: https://github.com/broadinstitute/gsa-unstable/pull/1574

**Review by kbergin — CHANGES_REQUESTED** [review]
Looking good! Very thorough. A few comments and suggestions based on other PR checklist

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (yfarjoun) [reply .github/PULL_REQUEST_TEMPLATE.md:13]: you can do this with template PR and Issue files (https://github.com/blog/2111-issue-and-pull-request-templates)

Comment (kbergin) [reply .github/PULL_REQUEST_TEMPLATE.md:13]: Thanks Yossi! I think that's what Khalid is doing here!

Comment (kshakir) [reply .github/PULL_REQUEST_TEMPLATE.md:13]: Copied and modified the header.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (kshakir) [reply .github/PULL_REQUEST_TEMPLATE.md]: Added a checkbox mentioning tests.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (kshakir) [reply .github/PULL_REQUEST_TEMPLATE.md:31]: 1. Talked to Henry. Clio will automatically move from dev to a release candidate in staging. There will be no need for a developer to manually update staging.
2. Fixed the last bullet point regarding marking the JIRA as resolved. Instead the ticket should be moved to QA.

**Comment by kshakir** [comment]
Ready for another round of reviews!

**Review by kbergin — APPROVED** [review]

**Comment by kbergin** [comment]
👍 !!

**Pull request merged**